### DISCUSSION TITLE
user의 password 유무

### DISCUSSION THREAD

**Discussion by Kimhan-nah** [GhostPangPang/GhostPong#90] [Q&A] [answered]
### 서브젝트 Security concerns에 `password`에 대한 언급
- Any password stored in your database must be hashed.
- Please make sure you use a strong password hashing algorithm

![image](https://user-images.githubusercontent.com/33301153/232401150-8598b650-c835-4d9a-92cb-c434c37facbb.png)

### 질문
- 로그인할 때 password를 추가해야 될까요...?
- 만약 추가한다면 user table에 password column이 추가 되어야 합니닷

### 참여자
@Kimhan-nah @jis-kim @anso33 @nyeoni @aLVINlEE9 
**!!천천히 의견 주셔도 됩니당!!**

**Comment by jis-kim** [comment, marked as answer]
이게 `user` 보다는 `channel` 을 타겟으로 하고 넣은 구문같은데 저희 로직 상 현재로는 비밀번호를 DB 에 저장할 일이 없어서 subject에 이런말이 있다고 해서 로그인에 password 를 끼워넣는 것은 불필요하다고 생각합니다. 그렇게 되면 로그인이 two-way 가 되어버리는데 이건 extended-feature 라서 지금은 42 로그인에만 집중하시면 될 것 같아요!

**Reply by Kimhan-nah** [reply]
오오 채널 비밀번호가 있었네요 완전히 이해되었씁니다! 이걸로 결정 탕탕탕!
